### PR TITLE
Add tests for signature schemes

### LINKED ISSUE #40: [Feature] Add tests for all signature schemes
Time estimation: 3 hours

### PR DESCRIPTION
**Pull request by jels-boolean**
Fixes #40 

Time taken: 3 hours

### CODE REVIEW

**Review by mowkoshka — CHANGES_REQUESTED** [review]

File: mentat/Cargo.toml [hunk 1]
```diff
@@ -41,12 +41,12 @@ mina-signer = { git = "https://github.com/o1-labs/proof-systems", optional = tru
 opentelemetry = { version = "0.17.0", optional = true, default-features = false, features = ["rt-tokio"] }
 opentelemetry-jaeger = { version = "0.16.0", optional = true, default-features = false, features = ["rt-tokio"] }
 p256 = { version = "0.10", optional = true, default-features = false }
-secp256k1 = { version = "0.22", optional = true, default-features = false }
+secp256k1 = { version = "0.22", optional = true }
 serde = { version = "1.0", optional = true, default-features = false, features = ["derive"] }
 serde_json = { version = "1.0", optional = true, default-features = false }
-snarkvm-algorithms = { version="0.8", optional = true, default-features = false }
```
[the remark this answers is not in the document]
Comment (gluax) [reply mentat/Cargo.toml]: My bad I never published this when I initially reviewed like a week ago :)

[the remark this answers is not in the document]
Comment (mowkoshka) [reply mentat/Cargo.toml]: My bad I never published this when I did the review a week ago :)

[the remark this answers is not in the document]
Comment (jels-boolean) [reply mentat/Cargo.toml]: I had a ton of compilation errors trying to get this to work and switching the features on actually made them disappear - I will look at what i can disable though.

[the remark this answers is not in the document]
Comment (jels-boolean) [reply mentat/Cargo.toml]: I've tested all crates, I could only remove default features for snarkvm algorithms and nothing else. These crates only get pulled in if needed through a feature flag anyways so it shouldn't be a big deal.

**Review by mowkoshka — APPROVED** [review]

**Pull request merged**